app/main.py

```diff
@@ -1,13 +1,16 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
+            "column" : error["column_number"],
```

Comment:
Code style issue: There is an extra space before the colon in the key-value pair. According to the checklist, variable names and formatting should be correct and consistent.